packages/react-core/src/components/Button/examples/ButtonProgress.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Button, Flex } from '@patternfly/react-core';
+
+export const ButtonProgress: React.FunctionComponent = () => {
+  const [isPrimaryLoading, setIsPrimaryLoading] = React.useState(true);
+  const [isSecondaryLoading, setIsSecondaryLoading] = React.useState(true);
+
+  return (
+    <Flex>
+      <Button
+        isLoading={isPrimaryLoading}
+        variant="primary"
+        onClick={() => setIsPrimaryLoading(!isPrimaryLoading)}
+        {...(isPrimaryLoading && { spinnerAriaValueText: 'Loading' })}
```

Comment:
This line is causing a problem when I hit the `ts>>js` button...

Reply:
The mechanism used to convert between TS and JS seems to break when spreading props (at least for Button). Can you point me to where we do the conversion so I can take a look?

Reply:
with about 70% confidence, I think that is what this line is supposed to do?
https://github.com/patternfly/patternfly-org/blob/main/packages/theme-patternfly-org/scripts/md/mdx-hast-to-jsx.js#L166